langchain/evaluation/agents/trajectory_eval_chain.py

```diff
@@ -69,7 +70,7 @@ def parse(self, text: str) -> TrajectoryEval:
         return TrajectoryEval(score=int(score_str), reasoning=reasoning)
 
 
-class TrajectoryEvalChain(Chain):
+class TrajectoryEvalChain(TrajectoryEvaluator, Chain):
```

Comment:
Will the final version of this class inherit from EvalChain instead of chain?

Reply:
Yes! Will update and diff